File: 0171-excel-sheet-column-number/0171-excel-sheet-column-number.py
class Solution:
    def getPos(self, c):
        return ord(c) - 64
    
    def titleToNumber(self, col: str) -> int:
        res = 0
        for i in range(len(col)):
            res += self.getPos(col[i]) * pow(26, len(col) - i - 1)
        
        return res

File: 0171-excel-sheet-column-number/0171-excel-sheet-column-number.js
var titleToNumber = function(columnTitle) {
  let k = 64;
  let columnNumber = 0;
  for (let i = 0; i < columnTitle.length; i++) {
    columnNumber += ((columnTitle.charCodeAt(i) - k) * Math.pow(26,columnTitle.length - i - 1));
  }
  return columnNumber;
};

